spread.yaml

```diff
@@ -444,6 +444,11 @@ debug-each: |
 
         echo '# journal messages for snapd'
         "$TESTSTOOLS"/journal-state get-log -u snapd
+
+        echo '# user sessions information'
+        journalctl --user -u snapd.session-agent.service
+        systemctl status --user snapd.session-agent  || true
```

Comment:
Keep in mind that wold print the session agent in the root's session, but at some point we may be interested in the sssion agent of the test user too.